src/github/nag.rs

```diff
@@ -578,6 +582,32 @@ fn parse_command_text<'a>(command: &'a str, subcommand: &'a str) -> &'a str {
 /// Parses all subcommands under the fcp command.
 /// If `fcp_context` is set to false, `@rfcbot <subcommand>`
 /// was passed and not `@rfcbot fcp <subcommand>`.
+/// 
+/// @rfcbot accepts roughly the following grammar:
```

Comment:
Could you update readme.md to explain this, since that file is linked to from rfcbot's PR comments?

Reply:
Yep; did that :)